netsim/ansible/templates/initial/ios.j2

```diff
@@ -79,7 +79,11 @@ interface {{ l.ifname }}
 #}
 {% if 'ipv4' in l %}
 {%   if l.ipv4 == True %}
+{%     if l.type is defined and l.type == "tunnel" %}
```

Comment:
By now, the l.type is defined for every interface (including the system loopback), so it's safe to use just `if l.type == "tunnel"` (similarly to what's done in some other parts of the same config template)

One of these days I have to clean up the old references, but there's always something better to do.